src/phoenix/server/api/schema.py

```diff
@@ -52,14 +65,16 @@ def projects(
             last=last,
             before=before if isinstance(before, Cursor) else None,
         )
-        data = (
-            []
-            if (traces := info.context.traces) is None
-            else [
-                Project(id_attr=project_id, name=project_name, project=project)
-                for project_id, project_name, project in traces.get_projects()
-            ]
-        )
+        async with info.context.db() as session:
+            projects = await session.scalars(select(models.Project))
+        data = [
+            Project(
+                id_attr=project.id,
+                name=project.name,
+                project=info.context.traces.get_project(project.name),  # type: ignore
```

Comment:
As a follow-up, it might make sense to key the core projects by ID rather than name.

Reply:
the core projects will be deleted after migration, so this is moot.